nimbleclient/v1/api/access_control_records.py

```diff
@@ -18,7 +20,11 @@ class AccessControlRecord(Resource):
     - chap_user_name       : Name of the CHAP user.
     - initiator_group_id   : Identifier for the initiator group.
     - initiator_group_name : Name of the initiator group.
-    - lun                  : If this access control record applies to a regular volume, this attribute is the volume's LUN (Logical Unit Number). If the access protocol is iSCSI, the LUN will be 0. However, if the access protocol is Fibre Channel, the LUN will be in the range from 0 to 2047. If this record applies to a Virtual Volume, this attribute is the volume's secondary LUN in the range from 0 to 399999, for both iSCSI and Fibre Channel. If the record applies to a OpenstackV2 volume, the LUN will be in the range from 0 to 2047, for both iSCSI and Fibre Channel. If this record applies to a protocol endpoint or only a snapshot, this attribute is not meaningful and is set to null.
+    - lun                  : If this access control record applies to a regular volume, this attribute is the volume's LUN (Logical Unit Number). If the access protocol is iSCSI,
+                             the LUN will be 0. However, if the access protocol is Fibre Channel, the LUN will be in the range from 0 to 2047. If this record applies to a Virtual
```

Comment:
Did this documentation come from the xml file?  If so, it's not quite accurate with group scoped target for iSCSI.  The LUN ID isn't always 0 in that case.

Reply:
Yes, these descriptions are fetched from the XML files.